examples/fluids/index.md

```diff
@@ -258,12 +258,12 @@ $$ (eq-test-perturbation-advdiff)
 
 See {cite}`hughesetal2010` equations 15-17 and 34-36 for further discussion of this formulation.
 
-For the Navier-Stokes and Euler equations in conservative variables, we follow {cite}`hughesetal2010` in defining a $3\times 3$ diagonal stabilization
-according to spatial criterion 2 (equation 27), whereas in the case of primitive variables, we follow {cite}`whiting2003hierarchical` in defining a $5\times 5$ diagonal stabilization consisting of
+For the Navier-Stokes and Euler equations in primitive variables, {cite}`whiting2003hierarchical` defines a $5\times 5$ diagonal stabilization consisting of
 1. continuity stabilization $\tau_c$
 2. momentum stabilization $\tau_m$
 3. energy stabilization $\tau_E$
 
+while we follow {cite}`hughesetal2010` in defining a $3\times 3$ diagonal stabilization according to spatial criterion 2 (equation 27), since our equations are in conservative form.
```

Comment:
Yeah, can you state the equation from Hughes2010 and how the eigenvalues of the flux Jacobian are the wave speeds `u-a, u, u+a`, because knowing this requires a different chunk of knowledge than what we're clearly referencing here.

For derivation, one can see, for example, lambda in Toro page 88-90.
![image](https://user-images.githubusercontent.com/3303/148088845-7ce8c6ff-75ae-4c29-aefb-4c8c1f096e56.png)
![image](https://user-images.githubusercontent.com/3303/148088888-6d3ac217-64da-46ee-86bb-980bd9e52a36.png)

Reply:
Sure, will do!